public/app/features/admin/ldap/LdapSettingsPage.tsx

```diff
@@ -199,6 +199,11 @@ export const LdapSettingsPage = () => {
         payload: [t('ldap-settings-page.alert.saved', 'LDAP settings saved')],
       });
       reset(await getSettings());
+
+      // Delay redirect so the form state can update
+      setTimeout(() => {
+        locationService.push(`/admin/authentication`);
+      }, 300);
```

Comment:
I don't like this, delaying an action by a random ms interval is usually not a good solution. Can you please investigate this issue and try to fix it?

Reply:
I don't like it either, but this is the same approach used thr Provider Config form

https://github.com/grafana/grafana/blob/3f6a64cc57b0a38b2c925f3d4f422c1ff42af5b4/public/app/features/auth-config/ProviderConfigForm.tsx#L93

Reply:
Removed as requested